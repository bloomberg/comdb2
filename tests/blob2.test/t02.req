insert into t2 values(NULL, x'abcdefabcdef', NULL)
insert into t2 values(NULL, x'abcdefabcdefabcdefabcdefabcdefabcdef', NULL)
exec procedure sys.cmd.verify('t2')
