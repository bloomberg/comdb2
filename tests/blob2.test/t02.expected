(rows inserted=1)
(rows inserted=1)
(out='Verify succeeded.')
